content/zh/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: blog
+title: "别慌: Kubernetes 和 Docker"
+date: 2020-12-02
+slug: dont-panic-kubernetes-and-docker
+---
+<!-- 
+layout: blog
+title: "Don't Panic: Kubernetes and Docker"
+date: 2020-12-02
+slug: dont-panic-kubernetes-and-docker
+-->
+
+**Authors:** Jorge Castro, Duffie Cooley, Kat Cosgrove, Justin Garrison, Noah Kantrowitz, Bob Killen, Rey Lejano, Dan “POP” Papandrea, Jeffrey Sica, Davanum “Dims” Srinivas
```

Comment:
if you want to, it's OK to localize “Authors” - see https://kubernetes.io/zh/blog/2020/12/18/kubernetes-1.20-pod-impersonation-short-lived-volumes-in-csi/ for an example.

Reply:
got it, fixed